content/en/llm_observability/setup/auto_instrumentation.md

```diff
@@ -137,12 +143,178 @@ The Google Gemini integration instruments the following methods:
 [13]: https://python.langchain.com/v0.2/docs/concepts/#llms
 [14]: https://python.langchain.com/v0.2/docs/concepts/#chat-models
 [15]: https://python.langchain.com/v0.2/docs/concepts/#runnable-interface
-[16]: /llm_observability/setup/sdk/
+[16]: /llm_observability/setup/sdk/python
 [17]: https://python.langchain.com/v0.2/docs/concepts/#embedding-models
 [18]: /llm_observability/setup/sdk/#tracing-spans
 [19]: https://ai.google.dev/gemini-api/docs
 [20]: https://ai.google.dev/api/generate-content#method:-models.streamgeneratecontent
+{{% /tab %}}
+{{% tab "Node.js" %}}
+
+## Overview
+
+Datadog's [LLM Observability Node.js SDK][4] provides integrations that automatically trace and annotate calls to LLM frameworks and libraries. Without changing your code, you can get out-of-the-box traces and observability for calls that your LLM application makes to the following frameworks:
+
+
+| Framework                               | Supported Versions | Tracer Version       |
+|-----------------------------------------|--------------------|----------------------|
+| [OpenAI](#openai) (common JS)           | >= 3.0.0           | >= 4.49.0, >= 5.25.0 |
+
+In addition to capturing latency and errors, the integrations capture the input parameters, input and output messages, and token usage (when available) of each traced call.
+
+## Enabling and disabling integrations
+
+All integrations are enabled by default.
+
+To disable all integrations, use the [in-code SDK setup][6] and specify `plugins: false` on the general tracer configuration.
+
+```javascript
+const tracer = require('dd-trace').init({
+  llmobs: { ... },
+  plugins: false
+});
+const { llmobs } = tracer;
+```
+
+To only enable specific integrations:
+1. Use the [in-code SDK setup][6], specifying `plugins: false`.
+2. Manually enable the integration with `tracer.use()` at the top of the entrypoint file of your LLM application:
+
+```javascript
+const tracer = require('dd-trace').init({
+  llmobs: { ... },
+  plugins: false
+});
+
+const { llmobs } = tracer;
+tracer.use('<INTEGRATION_NAME_IN_LOWERCASE>', true);
+```
+
+Additionally, you can set the following environment variables for more specific control over library patching and the integration that starts the span:
+
+`DD_TRACE_DISABLED_PLUGINS`
+: **Example**: `DD_TRACE_DISABLED_PLUGINS=openai,http`<br>
+A comma-separated string of integration names automatically disabled when the tracer is initialized.
+
+`DD_TRACE_DISABLED_INSTRUMENTATIONS`
+: **Example**: `DD_TRACE_DISABLED_INSTRUMENTATIONS=openai,http`<br>
+A comma-separated string of library names that are not patched when the tracer is initialized.
+
+## OpenAI
+
+The OpenAI integration provides automatic tracing for the [OpenAI Node.js SDK's][1] completion, chat completion, and embeddings endpoints.
+
+### Traced methods
+
+The OpenAI integration instruments the following methods, including streamed calls:
+
+- [Completions][2]:
+  - `openai.completions.create()`
```

Comment:
IMO it's overkill for us to list out each method we instrument. I don't think customers will care much and just want it to work :tm:

Given that this is already a pattern I'm ok to leave as-is and we can go back over everything and omit them in a follow up.

Reply:
yeah that makes sense to me. happy to follow up with it separately so we're consistent for python and node.js